austin_parham/dlists.py

```diff
@@ -0,0 +1,53 @@
+class Node:
+	def __init__(self,value):
+		self.value = value
+		self.next = None
+		self.previous = None
+
+
+class SList:
+	def __init__(self,value):
+		node = Node(value)
+		self.head = node
+
+	def addNode(self,value):
+		node = Node(value)
+		runner = self.head
+		while runner.next != None:
+			runner = runner.next
+		runner.next = node
+		runner.next.previous = runner
+		return self
+		# print(runner.next.previous.value)
+
+	def display_all(self):
+		runner = self.head
+		while runner.next != None:
+			print(runner.value)
+			runner = runner.next
+		print(runner.value)
+
+	def removeNode(self,value):
+		runner = self.head
+		if runner.value == value: #If the value is the first node
+			self.head = runner.next
+			runner.next.previous = None
```

Comment:
return after the if condition will make sure that nothing is executed after the first node is removed